README.md

```diff
@@ -51,7 +51,7 @@ In addition to the standard SublimeLinter settings, SublimeLinter-contrib-eslint
 |:------|:----------|:------------:|:-------------:|
 |eslint_d|runs "[eslint_d][eslint_d]" instead of "eslint"| | |
 
-[eslint_d][eslint_d] is a tool to reduce `eslint` startup time by running a server in the background. Now it works only with saved content of file, so probably you should set `Lint Mode` of SublimeLinter to `Load/save` or `Save only`. You must have `eslint_d` installed globally or locally with `npm install eslint_d`.
+[eslint_d][eslint_d] is a tool to reduce `eslint` startup time by running a server in the background. You must have `eslint_d` installed globally or locally with `npm install eslint_d`.
```

Comment:
@roadhump you sure it works with local eslint_d ? I'm not sure to see how if it's not in the user path.